src/utils.rs

```diff
@@ -59,7 +59,7 @@ pub fn get_command_string_output(command: CommandOutput) -> String {
 
 /// Attempt to resolve `binary_name` from and creates a new `Command` pointing at it
 /// This allows executing cmd files on Windows and prevents running executable from cwd on Windows
-/// This function also initialises std{err,out,in} to protect against processes changing the console mode
+/// This function also initializes std{err,out,in} to protect against processes changing the console mode
```

Comment:
This and the other s/z change are actually valid in British English, but I think it's fine because this works for both.